mypy/test/testcheck.py

```diff
@@ -173,33 +182,36 @@ def run_case_once(self, testcase: DataDrivenTestCase, incremental: int = 0) -> N
         a = normalize_error_messages(a)
 
         # Make sure error messages match
-        if incremental == 0:
+        if incremental_step == 0:
+            # Not incremental
             msg = 'Invalid type checker output ({}, line {})'
```

Comment:
Maybe change "Invalid" to "Unexpected" in each branch? This wording has always confused me.